released/discovery_center/mission_4172/main.tf

```diff
@@ -88,7 +88,6 @@ resource "btp_subaccount_entitlement" "name" {
 ######################################################################
 # Create service instances (and service keys when needed)
 ######################################################################
-
 # hana plan id
 data "btp_subaccount_service_plan" "hana_plan" {
```

Comment:
The data reads won't work after the timeout is removed. It makes sense to put the reads and the dependent resources into modules and use them in the main.tf

Reply:
If I put dependency on the entitlements, it works fine for me, tested at least 4 or 5 times this scenario... But if there is any special case, then I will encapsulate this in a separate module... Please let me know..